datasets/gov/nzt/matininkai.csv

```diff
@@ -5,16 +5,27 @@ id,dataset,resource,base,model,property,type,ref,source,prepare,level,access,uri
 ,,,,,,,vadovas,,,,,https://atviriduomenys.readthedocs.io/,,
 ,,,,,,,,,,,,,,
 ,,,,MatininkoPazymejimas,,,nr,,,4,,,"Matininkai, geodezininkai ir žemėtvarkos planavimo dokumentų rengėjai",
-,,,,,vardas_pavarde,string,,,,2,public,,"Matininko vardas, pavardė",Pateikta tik galiojantiems pažymėjimams
+,,,,,vardas_pavarde,string,,,,4,public,,"Matininko vardas, pavardė",Pateikta tik galiojantiems pažymėjimams
+,,,,,darbo_el_pastas,string,,,,4,public,,Matininko darbo el. paštas,"Pateikta tik galiojantiems pažymėjimams, negaliojantiems vertė cenzūruojama"
+,,,,,biuro_adresas,string,,,,2,public,,Matininko biuro adresas,"Pateikta tik galiojantiems pažymėjimams, negaliojantiems vertė cenzūruojama"
+,,,,,darbo_telefonas,string,,,,2,open,,Matininko dabo telefonas,"Pateikta tik galiojantiems pažymėjimams, negaliojantiems vertė cenzūruojama"
```

Comment:
```suggestion
,,,,,darbo_telefonas,string,,,,2,open,,Matininko darbo telefonas,"Pateikta tik galiojantiems pažymėjimams, negaliojantiems vertė cenzūruojama"
```
Kodėl nurodote  brandos lygį: ar dėl to, kad telefono numeris pateikiamas nestandartiniu formatu?